node/resolvers/headers.ts

```diff
@@ -27,6 +27,14 @@ export const withAuthToken = (currentHeaders = {}) => (ioContext, cookie = null)
   }
 }
 
+export const withAuthAsVTEXID = (currentHeaders = {}) => (ioContext) => {
+  return {
+    ...currentHeaders,
+    VtexIdclientAutCookie: ioContext.authToken,
```

Comment:
This is the admin token; Is it your intention?

Reply:
@brunoabreu what you think?

Reply:
Actually, this is the app token. We need to send the app token using `VtexIdclientAutCookie` header, because that's the way VTEX platform understand it.

In some cases, mainly when the resolver is meant to be used in admins, we use the `VtexIdclientAutCookie` cookie instead of `authToken`, because the user is supposed to be authenticated as administrator. But here the user is just a customer trying to get/save his profile data, then we have to use the app credentials for that.